docs/react-testing-library/setup.mdx

```diff
@@ -491,3 +491,72 @@ And register it using mocha's `-r` flag:
 ```
 mocha -r ./mocha-watch-cleanup-after-each.js
 ```
+
+### Vitest
+
+When using the react-testing-library with [Vitest](https://vitest.dev/),
+you need to install jsdom and the @testing-library:
+
+```bash npm2yarn
+npm install --save-dev vitest jsdom
+npm install --save-dev @testing-library/react @testing-library/jest-dom
+```
+
+Since the testing library uses jsdom, you must set the `environments`
+to `jsdom`.
+
+```js
+// vite.config.js
+export default defineConfig({
+  test: {
+    // highlight-next-line
+    environment: "jsdom",
+  },
+})
+```
+
+After installing the packages and setting the environment, you can start
+testing. However, some further steps might help you develop more efficiently.
+
+When utilizing testing-library APIs, you must first import
+`@testing-library/jest-dom` in every file. It's quite inconvenient.
```

Comment:
For `/react` you don't **need** `@testing-library/jest-dom`. Are we also sure `@testing-library/jest-dom` library even works Vitest?

Reply:
Sorry, I am not sure if this is what you are asking. If we set the `globals` to `true`, then the matchers will be automatically extended when importing `@testing-library/jest-dom`. However, we must extend the matchers manually if we do not set it to `true`.

Reply:
Sure but this document is about `@testing-library/react` not `@testing-library/jest-dom`. Let's split this out into a separate PR since I'm not sure jest-dom wants to support Vitest.